src/main/kotlin/model/FilledElements.kt

```diff
@@ -0,0 +1,94 @@
+package model
+
+import model.minemark.MineMark
+
+class FilledElements(elements: Map<Position, MineMark>) {
+
+    init {
+        require(isAllFilled(elements)) {
```

Comment:
이 클래스 뿐이 아닌 전체적으로 함수의 선언 위치를 조정해주시면 코드를 읽기 더 좋을것 같습니다.

https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
함수 선언 위치라는게 어떤 말씀이신지 잘 모르겠습니다 🤔

첨부해주신 링크를 살펴보면 다음과 같은 순서라고 표현되어 있는데 
혹시 어떤 순서가 잘못되었는지 알 수 있을까요???🤔🤔

Property declarations and initializer blocks
Secondary constructors
Method declarations
Companion object

Reply:
> put related stuff together, so that someone reading the class from top to bottom can follow the logic of what's happening. Choose an order (either higher-level stuff first, or vice versa) and stick to it.

위 부분을 말씀드린 것이었습니다.

함수들의 선언 위치가 섞여있는것으로 보입니다.